README.md

```diff
@@ -21,34 +21,19 @@ finding something going wrong. There's a long list of known issues
 Check out [the wiki page](https://github.com/new-xkit/XKit/wiki/Writing-a-New-Extension)!
 
 ## Develop XKit
-Download [Node.js](https://nodejs.org/download/) for your platform.
+Review the project prerequisites and learn how to build XKit from source [over on the wiki](/wiki/Build-XKit).
```

Comment:
Yup, you're right.  I'd forgotten how they mix in `/blob/master` to URLs like that.  I was hoping to make it relative to the project so the link would take users to a fork's wiki instead of the upstream one.

D'you think `../../wiki/Build-XKit` would be okay, or should we just go with an absolute URL?